app/views/layouts/_messages.html.slim

```diff
@@ -1,12 +1,11 @@
 - unless flash.empty?
-  .mb1
```

Comment:
removed unnecessary flash messages container div (and added check for non-blank messages)